examples/assemblyscript/assemblyscript.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/tetratelabs/wazero"
+	"github.com/tetratelabs/wazero/assemblyscript"
+)
+
+// asWasm compiled using `npm install && npm run build`
+//go:embed testdata/assemblyscript.wasm
+var asWasm []byte
+
+// main shows how to interact with a WebAssembly function that was compiled
+// from AssemblyScript
+//
+// See README.md for a full description.
+func main() {
+
+	// Choose the context to use for function calls.
+	ctx := context.Background()
+
+	// Create a new WebAssembly Runtime. AssemblyScript enables bulk-memory operations by default
+	// so we enable it in the runtime too.
+	r := wazero.NewRuntimeWithConfig(wazero.NewRuntimeConfigCompiler().
```

Comment:
I think this is the root cause of the failure :D 
```suggestion
	r := wazero.NewRuntimeWithConfig(wazero.NewRuntimeConfig().
```

Reply:
Oops, thanks!